Makefile

```diff
@@ -48,7 +48,8 @@ ifeq ($(ENABLE_UNLOCK),true)
 	BASE_CMD += --allow-insecure-unlock
 endif
 ifeq ($(BOOTNODE),true)
-	BASE_CMD += --nodekey bootnode.key --ws.origins=$(WS_ORIG) --http.corsdomain=$(HTTP_CORSDOMAIN)
+	EXT_IP = $(shell curl -s ifconfig.me)
```

Comment:
If its only used for bootnode, we should hard code this value. I don't see a point in using curl like this, no need to rely on any service, since we anyways manually update the bootnodes.

Reply:
I've removed the dynamic setting of the EXT_IP in favor of manually setting it in network.env